File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split(" ");
        for(int i = 0; i < str.length; i++) {
            String ss = str[i];
            //공백 문자일시
            if(str[i].length() == 0) {
                answer += " ";
            }
            else {
                answer += ss.substring(0,1).toUpperCase();
                answer += ss.substring(1).toLowerCase();
                answer+=" ";
            }
        }
        if(s.substring(s.length()-1, s.length()).equals(" ")){
    		return answer;
    	}
        
        
        
        return answer.substring(0,answer.length()-1);
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    s = s.toLowerCase();
    const ans = [];
    const arr = s.split(' ');
    for (let i = 0; i < arr.length; i++) {
        let str = '';
        for (let j = 0; j < arr[i].length; j++) {
            console.log(arr[i][j]);
            if (j ===0 && 'a' <= arr[i][j] && arr[i][j] <= 'z') str += arr[i][j].toUpperCase();
            else str += arr[i][j];
            
        }
    ans.push(str);
    }
    answer = ans.join(' ');
    return answer;
}